README.md

```diff
@@ -88,34 +106,68 @@ yum install -y epel-release
 yum install -y readline-devel leveldb-devel libffi-devel gcc-c++ redhat-rpm-config gcc python-devel openssl-devel
 ```
 
-### For all of these, make sure that the `Chains` directory in your project has the proper write permissions
+#### Windows
 
-##### Windows
+Help needed. Installing the Python package plyvel seems to require C++ compiler
+support tied to Visual Studio and libraries.
 
-Not sure. Installing the Python package plyvel seems to require C++ compiler support tied to Visual Studio and libraries.
+### Python 3.5
 
--------------------
+neo-python is currently only compatible with **Python 3.5** (due to its `byteplay3` dependency).
 
-### Virtual Environment
+On *nix systems, we can easily install more Python versions on one computer
+using [pyenv](https://github.com/pyenv/pyenv).
```

Comment:
Not sure we gonna recommend `pyenv` as default tool. Perhaps we talk about this in a separate thread. Installing Python 3.5 on *nix is easy enough with the system package manager?

Reply:
I've changed the wording, I think it's a good tip for people who don't want to mess up their system, but I agree, that way it sounds like a precondition which is not the case.